components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -200,17 +217,13 @@ GURL AssetRatioService::GetSardineBuyURL(const std::string& chain_id,
 }
 
 // static
-GURL AssetRatioService::GetPriceURL(const std::vector<std::string>& from_assets,
-                                    const std::vector<std::string>& to_assets,
-                                    mojom::AssetPriceTimeframe timeframe) {
-  std::string from = VectorToCommaSeparatedList(from_assets);
-  std::string to = VectorToCommaSeparatedList(to_assets);
-  std::string spec =
-      absl::StrFormat("%s/v2/relative/provider/coingecko/%s/%s/%s",
-                      base_url_for_test_.is_empty() ? GetAssetRatioBaseURL()
-                                                    : base_url_for_test_.spec(),
-                      from, to, TimeFrameKeyToString(timeframe));
-  return GURL(spec);
+GURL AssetRatioService::GetPriceURL(const std::string& vs_currency) {
+  std::string base_url =
+      base_url_for_test_.is_empty() ? kGate3URL : base_url_for_test_.spec();
+  GURL url = GURL(absl::StrFormat("%s/api/pricing/v1/getPrices", base_url));
+  url = net::AppendQueryParameter(url, "vs_currency",
+                                  base::ToUpperASCII(vs_currency));
+  return url;
 }
 
```

Comment:
Should we do something with `GetPriceHistoryURL` below?

Reply:
Price history support is not available on gate3 yet. For the time being, we'll continue using Ratios for this.